### ISSUE TITLE
loss and val_loss didnt change during training

### ISSUE THREAD

**Issue by royxue** [fchollet/keras#6132]
I was training a neural network, somehow, during my training, the loss and val_loss values remains the same. 
`
Epoch 1/5
52/51 [==============================] - 1423s - loss: 1.0097 - val_loss: 1.0082
Epoch 2/5
52/51 [==============================] - 1417s - loss: 1.0097 - val_loss: 1.0082
Epoch 3/5
52/51 [==============================] - 1417s - loss: 1.0097 - val_loss: 1.0082
`

Here is my mode compile and fit_generator code, is there anything I missed?
`model.compile(loss='mse', optimizer='adam')`
`train_steps = len(train)/NUM_BATCH`
`valid_steps = len(valid)/NUM_BATCH`
`metrics = model.fit_generator(train_gen, steps_per_epoch=train_steps, `
`                    validation_data=valid_gen,validation_steps=valid_steps, epochs=EPOCHS, verbose=1)`

**Comment by royxue** [comment]
Solved by myself.
Seems Keras 2.0 removed the sub_sample from Conv2D
Seems if you want to do MaxPooling and then Relu
You should add Relu after MaxPooling2D instead of using activation='relu' in Conv2D
